src/applications/_mock-form-ae-design-patterns/App.jsx

```diff
@@ -1,43 +1,10 @@
-import React, { useEffect } from 'react';
-
-import { useDispatch } from 'react-redux';
+import React, { useContext, useEffect } from 'react';
 
 import RoutedSavableApp from 'platform/forms/save-in-progress/RoutedSavableApp';
 import FormFooter from 'platform/forms/components/FormFooter';
 
-// removes local storage for user profile session indicators
-
-import { teardownProfileSession } from 'platform/user/profile/utilities';
-
-import greenFormConfig from './patterns/pattern1/TaskGreen/config/form';
-import yellowFormConfig from './patterns/pattern1/TaskYellow/config/form';
-import purpleFormConfig from './patterns/pattern1/TaskPurple/config/form';
-import ezrFormConfig from './patterns/pattern1/ezr/config/form';
-import { useLocalStorage } from './hooks/useLocalStorage';
-import { TaskTabs } from './shared/components/TaskTabs';
-import { Portal } from './shared/components/Portal';
 import { LOCATIONS_TO_REMOVE_FORM_HEADER } from './utils/constants';
-import fallbackForm from './shared/config/fallbackForm';
-
-export const getFormConfig = location => {
-  if (location.pathname.includes('task-green')) {
-    return greenFormConfig;
-  }
-
-  if (location.pathname.includes('task-yellow')) {
-    return yellowFormConfig;
-  }
-
-  if (location.pathname.includes('task-purple')) {
-    return purpleFormConfig;
-  }
-
-  if (location.pathname.includes('ezr')) {
-    return ezrFormConfig;
-  }
-
-  return fallbackForm;
-};
+import { PatternConfigContext } from './shared/context/PatternConfigContext';
```

Comment:
By using our `PatternConfigContext` the main logic around loading configurations for forms is abstracted away, and we can now use different `App` components when needed and wrap them with the context provider